.github/workflows/main.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
         run: |
           rsync -r --exclude ./lib ./lib/* .
-          npx -p semantic-release -p @semantic-release/git -p @semantic-release/changelog -p @semantic-release/exec -p semantic-release-github semantic-release --provider=github --debug=true
+          npx -p pinst -p semantic-release -p @semantic-release/git -p @semantic-release/changelog -p @semantic-release/exec -p semantic-release-github semantic-release --provider=github --debug=true
```

Comment:
why do we need pinst now?

Reply:
Requires Huksy for a package installation (https://typicode.github.io/husky/#/?id=install-1). 

`postinstall` must be ignored because of the Husky installation and Semantic-release executes `prepublishOnly` and `postpublish`.

![image](https://user-images.githubusercontent.com/8287751/113434308-fa866080-93e0-11eb-9122-fd674ae2950c.png)